exp_final_whitebox_attacker.py

```diff
@@ -0,0 +1,53 @@
+import sys
+
+from gbra.data.network_loader import Movielens100kLoader
+from gbra.attackers.attacker import *
+from gbra.recommender.recommenders import PixieRandomWalkRecommender
+
+ITERATIONS = 10
+
+PIXIE_PARAMS = {
+    'n_p': 30,
+    'n_v': 4,
+    'num_steps_in_walk': 200,
+    'alpha': 0.01
+}
+RECOMMENDATIONS = 10
+
+""""
+python exp_final_whitebox_attacker.py 0.10 10 HighDegreeAttacker
+"""
+
+PERCENT_FAKE_ENTITIES, NUM_FAKE_REVIEWS, ATTACKER_NAME = sys.argv[1:]
+PERCENT_FAKE_ENTITIES = float(PERCENT_FAKE_ENTITIES)
+NUM_FAKE_REVIEWS = int(NUM_FAKE_REVIEWS)
+
+attackers = {
+    'HighDegreeAttacker': HighDegreeAttacker,
+    'LowDegreeAttacker': LowDegreeAttacker,
+    'HillClimbingAttacker': HillClimbingAttacker
+}
+
+def get_attacker(network, recommender, target_item):
+    attacker_klass = attackers[ATTACKER_NAME]
+    num_fake_entities = int(percent_fake_entities * network.num_entities)
+    return attacker_klass(recommender, target_item, num_fake_entities, NUM_FAKE_REVIEWS)
+
+def evaluate_attacker():
+    network = Movielens100kLoader().load()
+
+    # Target item to popularize
+    [target_item] = network.get_random_items(1)
```

Comment:
Instead of getting a random item could we choose a fixed and relatively unpopular item?  We should look at the same item across all attackers and experiments.

Reply:
(this is OK because we are sampling across 10 iterations)